bin/pycbc_multi_inspiral

```diff
@@ -972,7 +974,10 @@ with ctx:
                                 coinc_idx_det_frame[ifo]
                                 + stilde[ifo].analyze.start,
                             )
-                            ifo_out_vals['cont_chisq'] = autochisq.values(
+                            (
+                                ifo_out_vals['auto_chisq'],
+                                ifo_out_vals['auto_chisq_dof'],
+                            ) = autochisq.values(
```

Comment:
Does this work? I'd expect `ifo_out_vals['auto_chisq_dof']` to be a single integer, and `ifo_out_vals['auto_chisq']` to be an array. Would this cause issues later if this isn't handled correctly?

... While I'm happy for this function to return the DOF parameter for consistency, I'm not sure it's going to be useful for much (assuming the auto_chisq value is already divided by DOF).

Reply:
Yes, it works, no there are no issues because the `numpy.repeat` call in this case is [here](https://github.com/gwastro/pycbc/blob/master/pycbc/events/eventmgr.py#L1035), which replicates [here](https://github.com/gwastro/pycbc/blame/master/pycbc/events/eventmgr.py#L499).
Let me know if you want me to move the call to from these two lines to `pycbc/vetoes/autochisq.py` in the spirit of make things even across the chi-squares.  I have to push another commit in any event because I noticed that [this description](https://github.com/gwastro/pycbc/blob/master/pycbc/vetoes/autochisq.py#L62) is incomplete.

Reply:
No, if there is a `numpy.repeat` then this is not a concern.